src/lib/common.js

```diff
@@ -101,10 +101,27 @@ const getConfigEnv = async options =>
     options
   )
 
+const sanitizeConfig = config => {
+  // recurse configuration objects and arrays to remove whitespace
+  const sanitize = (chunk) => {
```

Comment:
Throughout, I opted to use lodash convenience methods since you had already included the package. If you'd prefer I use native methods (e.g. `Array.isArray`, native `.trim()`) let me know and I can swap them out!